src/integration/pages/articles/news/__snapshots__/amp.test.js.snap

```diff
@@ -266,6 +266,8 @@ exports[`AMP Articles Image should match snapshot 1`] = `
 
 exports[`AMP Articles Main heading should match text 1`] = `"Royal wedding flowers given to Hackney hospice"`;
 
+exports[`AMP Articles Paragraph should match paragraph text 1`] = `"We've made some important changes to our Privacy and Cookies Policy and we want you to know what this means for you and your data."`;
```

Comment:
This is coming from the cookie banner... should we update the paragraph test to check within `main`

Reply:
done